### ISSUE TITLE
Accept input via command chaining

### ISSUE THREAD

**Issue by jthurman42** [newrelic/newrelic-cli#45]
### Feature Description
Add the ability to chain newrelic commands together, example would be an entity search filtered to GUIDs, then piped back to newrelic to add tags to all of those entities.

```
$ newrelic entity search --name "mystuff" --only-attributes entityId | \
  newrelic entity tags add --tag owner:me
```

**Comment by nicolasjhampton** [comment]
@jthurman42 I intend to start tackling this issue, using the entity command as an MVP, but it would help if you could answer a question. This example code includes a secret request to add a feature for bulk edits. Currently in the cli, there's no way to specify multiple entities to act on at once. Should I aim here to add that functionality?

**Comment by jthurman42** [comment]
@nicolasjhampton Good point, there is an assumption that the CLI would handle an array of input items and loop thought them based on the example.  I believe my thought here was that the data passed through the pipe would be in JSON, so easy enough to detect an array to iterate on.  I'd prefer to see that at least though of, otherwise we're pushing the work of splitting / running multiple items out to the user.

**Comment by nicolasjhampton** [comment]
@jthurman42 I've gotten all the way to getting bulk information from stdin to the actual newrelic client query, but there's literally not a method on the newrelic-client-go package that takes several guids and returns the tags for all of them. 

Like we agreed before, this is actually two legs of work. I think I'm complete on getting bulk data from stdin and making it accessible to each command to use. If I can get your signoff for this to be reviewed and merged separately, I'll do an inventory of what needs to change in newrelic-client-go to allow useful bulk actions and make a PR there to make further work here possible.

**Comment by Julien4218** [comment]
cc @kidk
